packages/teams-js/src/private/interfaces.ts

```diff
@@ -132,6 +132,15 @@ export interface FilePreviewParameters {
    */
   type: string;
 
+  /**
+   * @hidden
+   * The size of the file in bytes.
+   *
+   * @internal
+   * Limited to Microsoft-internal use
+   */
+  size?: string;
```

Comment:
@TrevorJoelHarris , is there any general guidance where we should put the parameter for new added one? The parameter list is not alphabet order so it doesn't look there is any pattern we can follow... maybe it doesn't matter.. The reason I asked is because sometimes I wondered the same when I make the similar changes.

Reply:
@TrevorJoelHarris can you please update on this.

Reply:
It doesn't matter very much, the order doesn't affect external developers in a meaningful way. It would be nice if they were alphabetized but I don't think @reenudeswal needs to do that in this PR.